phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/ContainerVerbsTab.java

```diff
@@ -0,0 +1,94 @@
+package org.phoenicis.javafx.views.mainwindow.containers;
+
+import javafx.application.Platform;
+import javafx.scene.Node;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.Tab;
+import javafx.scene.layout.TilePane;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Text;
+import org.phoenicis.containers.dto.ContainerDTO;
+import org.phoenicis.engines.VerbsManager;
+import org.phoenicis.javafx.views.common.ErrorMessage;
+import org.phoenicis.javafx.views.common.TextWithStyle;
+import org.phoenicis.repository.dto.ApplicationDTO;
+import org.phoenicis.repository.dto.ScriptDTO;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.phoenicis.configuration.localisation.Localisation.tr;
+
+public class ContainerVerbsTab extends Tab {
+    private static final String CONFIGURATION_PANE_CSS_CLASS = "containerConfigurationPane";
+    private static final String TITLE_CSS_CLASS = "title";
```

Comment:
Can we directly use the css classes in the files please?
I'm not a huge fan of global variables for css classes

Reply:
Done.